### ISSUE TITLE
failed to join zone error

### ISSUE THREAD

**Issue by scroat** [South-Paw/salien-script-js#49]
total noob here, followed instructions on https://github.com/South-Paw/salien-script-js and script ran perfectly for an hour or so.

then it got failed to join error and "restart in 5 seconds" loop.

says it is trying to join Zone 0 on planet 38 each time.

have updated 3 times. have tried install command after update. same result. any info for this admittedly clueless old me?

**Comment by CrazyCoder** [comment]
Same issue, started to happen just recently.

**Comment by holamlui** [comment]
Well looks like there's an error with this Zone, even if I manually join this zone in the browser it didn't work, the script may need an update for skipping dead zones

**Comment by scroat** [comment]
interesting! is there any way to manually skip the zone?

**Comment by cheesestringer** [comment]
In game/index.js you can just add the below to ```planet.zones.forEach``` on line 82 as a temporary workaround.
```
const { capture_progress: captureProgress, captured, type, difficulty, zone_position: zonePosition } = zone;
    if(zonePosition === 0){
        return;
    }
```

**Comment by scroat** [comment]
thanks cheesestringer.

could you tell me where i find game/index.js? i tried searching within nodejs but didn't come up with anything.

**Comment by cheesestringer** [comment]
Should be something like: ```C:\Users\YOUR_USERNAME\AppData\Roaming\npm\node_modules\salien-script-js\src\game```

**Comment by holamlui** [comment]
@cheesestringer 
I think it should be `zone.zone_position ` instead of just `zone_position `

**Comment by scroat** [comment]
thanks guys @holamlui @cheesestringer, found it.

looking for best prog to open it now :P

**Comment by cheesestringer** [comment]
@holamlui at the end of the first line ```zonePostion``` is mapped to ```zone_position```.

@scroat you can edit it in notepad.

**Comment by scroat** [comment]
@cheesestringer i went there first but it didn't index the lines a-la "line 82" etc so i thought some other program must have been used. 

when i open it in notepad its just a huge chunk of text.

**Comment by holamlui** [comment]
@cheesestringer you're right, I didn't see you add the mapping for zonePosition in the code

**Comment by holamlui** [comment]
@cheesestringer you're right, I miss the mapping for zonePosition you added in the code

**Comment by holamlui** [comment]
@scroat It's not really a good idea to open code using notepad, 
at least use notepad++ for a better view

**Comment by Rideu** [comment]
@scroat just use CTRL + F to search for this variable. Also you can use Notepad++ to simplify the process in future.

**Comment by South-Paw** [comment]
Fixed with with #50 courtesy of @solareon 👍

**Comment by scroat** [comment]
thanks for all the help guys.
